curriculum/challenges/english/25-front-end-development/lab-lunch-picker-program/66db529d37ad966480ebb633.md

```diff
@@ -0,0 +1,149 @@
+---
+id: 66db529d37ad966480ebb633
+title: Build a Lunch Picker Program
+challengeType: 14
+dashedName: build-a-lunch-picker-program
+---
+
+# --description--
+
+In this lab, you will create a program that will help you decide what to eat for lunch. You will have two arrays with different food items. You will use the `push`, `pop` and `shift` methods to modify the arrays. You will also use the `Math.random()` method to generate a random number that will be used to select a random meal from the arrays.
+
+**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab. 
+
+**User Stories:**
+
+1. You should create a `kidsMeal` array and assign more than two string values to it. You can either use `push()` or assignment to add the items to the array.
```

Comment:
we shouldn't mention how they should go about it. it is best to have campers think a little about what method or approach should work here.

```suggestion
1. You should create a `kidsMeal` variable and assign it an array with at least three strings in it.
```